basics.tex

```diff
@@ -1276,7 +1276,7 @@ \section{Cartesian product types}
   Now by induction on $p$ and $q$, we may assume they are reflexivities $\refl a$ and $\refl b$, in which case~\eqref{eq:path-prod-inverse} yields $\refl{(a,b)}$ and then~\eqref{eq:path-prod} returns us to $(\refl a,\refl b)\jdeq (p,q)\jdeq s$.
 \end{proof}
 
-In particular, we have shown that~\eqref{eq:path-prod} has an inverse~\eqref{eq:path-prod-inverse}, which we may denote by
+In particular, we have shown that~\eqref{eq:path-prod} has a quasi-inverse~\eqref{eq:path-prod-inverse}, which we may denote by
```

Comment:
Hmm.  We defined "a quasi-inverse" to be a function in the other direction *together with* two homotopies.  I would have thought it's okay to say "inverse" when referring only to the inverse function.